<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Git::Internal::SettingsPage</class>
 <widget class="QWidget" name="Git::Internal::SettingsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>403</width>
    <height>251</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="environmentGroupBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Environment variables</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="pathlabel">
          <property name="text">
           <string>PATH:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="pathLineEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="adoptButton">
          <property name="text">
           <string>From system</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="noteLabel">
          <property name="text">
           <string>&lt;b&gt;Note:&lt;/b&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="noteFieldlabel">
          <property name="text">
           <string>Git needs to find Perl in the environment as well.</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="logCountLabel">
       <property name="text">
        <string>Log commit display count:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="logCountSpinBox">
       <property name="toolTip">
        <string>Note that huge amount of commits might take some time.</string>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="timeoutLabel">
       <property name="text">
        <string>Timeout (seconds):</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="timeoutSpinBox">
       <property name="minimum">
        <number>10</number>
       </property>
       <property name="maximum">
        <number>300</number>
       </property>
       <property name="value">
        <number>30</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>pathLineEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
